src/group/dto/update-group.dto.ts

```diff
@@ -1,4 +1,12 @@
 import { PartialType } from '@nestjs/mapped-types';
 import { CreateGroupDto } from './create-group.dto';
 
-export class UpdateGroupDto extends PartialType(CreateGroupDto) {}
+export class UpdateGroupDto extends PartialType(CreateGroupDto) {
+  id?: string;
+  danceType: string;
+  advanceLevel: string;
+  teacherId: string[];
+  studentId: string[];
+  maxAmount: number;
+  scheduleId: number;
+}
```

Comment:
W dokumentacji wynika że samo rozszerzenie extends PartialType(CreateGroupDto) mowi nam o tym że UpdateGroupDTO nie musi miec żadnych parametrów.

Reply:
czyli do usunięcia pójdą w takim razie linijki od 5 do 11? będzie pusty obiekt? Chyba jednak trzebaby dodać to `id` chociażby? @DariuszKny